proto/proto/note.proto

```diff
@@ -22,6 +22,7 @@ message NoteSyncRecord {
     digest.Digest note_id = 2;
     account.AccountId sender  = 3;
     fixed64 tag = 4;
+    note.NoteType note_type = 5;
     merkle.MerklePath merkle_path = 6;
 }
```

Comment:
btw, while we re at it: the `tag` is now a `u32` value - so, we should probably use `fixed32` for it (here and in other messages). We could change it in a different PR though (and we'll need to handle 32-bit tags on the client as well).

Reply:
Done. We have to deal with the other changes from the client anyway.